src/tools/rbd/Utils.cc

```diff
@@ -762,5 +762,93 @@ std::string timestr(time_t t) {
   return buf;
 }
 
+int export_diff(librbd::Image& image, const char *fromsnapname,
```

Comment:
I'd rather see a new file for the main export/import diff logic instead of putting it into Utils (for lack of a better place).